x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/classification_exploration/column_data.tsx

```diff
@@ -4,79 +4,101 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import React from 'react';
-import { ConfusionMatrix, PredictedClass } from '../../../../common/analytics';
+import React, { useState } from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiButtonEmpty, EuiButtonIcon, EuiPopover, EuiText } from '@elastic/eui';
+import { ConfusionMatrix } from '../../../../common/analytics';
 
 interface ColumnData {
   actual_class: string;
   actual_class_doc_count: number;
-  predicted_class?: string;
-  count?: number;
-  error_count?: number;
+  [key: string]: any;
```

Comment:
Would `[key: string]: string | number;` work?

Reply:
Yeah - good catch. Added in 422c7aacdfc9d129a7877af2d9f1c8ebede15f80